test/integration/reachability.go

```diff
@@ -43,19 +43,22 @@ func (r *reachability) run() error {
 		r.accessLogs[app] = make([]string, 0)
 	}
 
-	err := r.makeRequests()
-	if err != nil {
+	if err := r.makeRequests(); err != nil {
 		return err
 	}
 	if verbose {
 		log.Println("requests:", r.accessLogs)
 	}
 
-	if err = r.checkAccessLogs(); err != nil {
+	if err := r.checkAccessLogs(); err != nil {
 		return err
 	}
 
-	if err = r.checkMixerLogs(); err != nil {
+	if err := r.checkMixerLogs(); err != nil {
+		return err
+	}
+
+	if err := r.verifyTCPRouting(); err != nil {
```

Comment:
We can do a check with stats or use mixer logs. Not necessary for now, just something to do later.